metaci/testresults/robot_importer.py

```diff
@@ -276,3 +307,37 @@ def render_robot_test_xml(root, test):
 
     test_xml = ET.tostring(testroot, encoding="unicode")
     return re.sub(r"sid=.*<", "sid=MASKED<", test_xml)
+
+
+def export_robot_test_results(flowtest, test_results) -> None:
+    if not settings.METACI_RELEASE_WEBHOOK_URL or not flowtest:
+        return  # should we better error handle this for individual case message error handling?
```

Comment:
```suggestion
        return
```

Also, how would it get called without a flowtask?